src/models/isbn-registry/validators.js

```diff
@@ -28,7 +28,7 @@
 import {calculateCheckDigitIsbn} from '../../interfaces/isbn-registry/ranges/isbnUtils';
 
 export function isValidIsbnOrIsmnIdentifier(value) {
```

Comment:
Valid isbn value schema will change over time. 
https://www.npmjs.com/package/isbn3 gets 1/month update on that list from [isbn-international.org](https://www.isbn-international.org/) would it be better to use it? or make own same style system?

Reply:
Good point. My initial thought is that no matter what we choose each solution includes a trade-off: making similar system takes effort but on the other hand having a external library taking care of data integrity in this case introduces a new risk factor. The current hardcoded system on the other hand fails when new ranges are introduced but should not otherwise introduce risks. Errors regarding failure of creating identifiers because of invalid range definition are the type that cannot go unnoticed and are easy to fix.

I think we have to evaluate further before making any decisions.